src/components/Comment.js

```diff
@@ -1,17 +1,40 @@
-import React from "react";
+import React, { useContext } from "react";
+import { UserContext } from "../contexts/User";
+import { deleteComment } from "../api";
+
+function Comment({ comment, setDeleted }) {
+  const { user } = useContext(UserContext);
+
+  const deleteHandler = (e) => {

```

Comment:
You'll need to add a way to prevent users from making multiple delete requests before the initial one has been completed (e.g. on slow connections). This should only trigger once to prevent 404 requests coming back from axios when attempting to delete an article that no longer exists.